package.json

```diff
@@ -85,7 +85,8 @@
     "@react-native-google-signin/google-signin": "^10.0.1",
     "@react-native-picker/picker": "2.5.1",
     "@react-navigation/material-top-tabs": "^6.6.3",
-    "@react-navigation/native": "6.1.8",
+    "@react-navigation/native": "6.1.12",
+    "@react-navigation/native-stack": "^6.9.17",
```

Comment:
I see that `native-stack` package was imported here. Is this being used anywhere?

Reply:
looks like it's not, so we can remove this

Reply:
Yeah, I must have made an error while merging main as native-stack was removed recently from the codebase